src/pineko/cli/fonll.py

```diff
@@ -0,0 +1,154 @@
+        for member in operand:
+            ret.append(f"{member}.{parser.EXT}")
+    return ret
+
+
+@command.command("combine_fonll")
+@click.argument("theoryID", type=int)
+@click.argument("dataset", type=str)
+@click.option("--FFNS3", type=int, help="theoryID containing the ffns3 fktable")
+@click.option("--FFN03", type=int, help="theoryID containing the ffn03 fktable")
+@click.option("--FFNS4til", type=int, help="theoryID containing the ffns4til fktable")
+@click.option("--FFNS4bar", type=int, help="theoryID containing the ffns4bar fktable")
+@click.option("--FFN04", type=int, help="theoryID containing the ffn04 fktable")
+@click.option("--FFNS5til", type=int, help="theoryID containing the ffns5til fktable")
+@click.option("--FFNS5bar", type=int, help="theoryID containing the ffns5bar fktable")
+@click.option("--overwrite", is_flag=True, help="Allow files to be overwritten")
+@config_setting
+def subcommand(
+    theoryid,
+    dataset,
+    ffns3,
+    ffn03,
+    ffns4til,
+    ffns4bar,
+    ffn04,
+    ffns5til,
+    ffns5bar,
+    overwrite,
+    cfg,
+):
+    """Combine the different FKs needed to produce the FONLL prescription."""
+    path = configs.detect(cfg)
+    base_configs = configs.load(path)
+    configs.configs = configs.defaults(base_configs)
+    if cfg is not None:
+        print(f"Configurations loaded from '{path}'")
+
+    # Checks
+    if (ffn04 is None and ffns5til is not None) or (
+        ffn04 is not None and ffns5til is None
+    ):
+        print(
+            "One between ffn04 and ffns5til has been provided without the other. Since they are both needed to construct FONLL, this does not make sense."
+        )
+        raise InconsistentInputsError
+    if (ffn03 is None and ffns4til is not None) or (
+        ffn03 is not None and ffns4til is None
+    ):
+        print(
+            "One between ffn03 and ffns4til has been provided without the other. Since they are both needed to construct FONLL, this does not make sense."
+        )
+        raise InconsistentInputsError
+    # Get theory info
+    tcard = theory_card.load(theoryid)
+    if not "DAMPPOWER" in tcard:
+        if tcard["DAMP"] != 0:
+            raise InconsistentInputsError
+        tcard["DAMPPOWER"] = None
+    # Getting the paths to the grids
+    grids_name = get_list_grids_name(
+        configs.configs["paths"]["ymldb"] / f"{dataset}.yaml"
+    )
+    for grid in grids_name:
+        # Checking if it already exists
+        new_fk_path = configs.configs["paths"]["fktables"] / str(theoryid) / grid
+        if new_fk_path.exists():
+            if not overwrite:
+                rich.print(
+                    f"[green]Success:[/] skipping existing FK Table {new_fk_path}"
+                )
+                return
+        fonll.produce_combined_fk(
+            configs.configs["paths"]["fktables"] / str(ffns3) / grid
+            if (configs.configs["paths"]["fktables"] / str(ffns3) / grid).exists()
+            else None,
```

Comment:
definitely too much repetition, I'd say
```py
def load_cfg(name):
    return configs.configs["paths"]["fktables"] / name / grid            if (configs.configs["paths"]["fktables"] / name / grid).exists()            else None
```
and then
```py
*(load_cfg(str(var)) for var in (ffns3, ffn03))
```

Reply:
Even better:
```py
def load_cfg(name):
    path = configs.configs["paths"]["fktables"] / name / grid
    return  path if path.exists() else None
```

Reply:
[c718a1f](https://github.com/NNPDF/pineko/pull/99/commits/c718a1f4c7b3d352c7ed365c7cac52196b00fe31)